app/templates/views/guidance/using-notify/attach-pages.html

```diff
@@ -0,0 +1,45 @@
+{% extends "content_template.html" %}
+
+{% from "components/service-link.html" import service_link %}
+
+{# Used by the content_template.html layout, prefixes the "navigation" accessible name #}
+{% set navigation_label_prefix = 'Using Notify' %}
+
+{% block per_page_title %}
+  Attach pages
+{% endblock %}
+
+{% block content_column_content %}
+
+  <h1 class="heading-large">Attach pages</h1>
+
+  <p class="govuk-body">Add attachments to your letters such as documents, tables and direct debit mandates.</p>
+  <p class="govuk-body">Upload a PDF and we’ll print it as part of your letter.</p>
```

Comment:
```suggestion
    <p class="govuk-body">Upload a PDF and we’ll print it as part of your letter.</p>
    <p class="govuk-body">You can attach pages with complex layouts. For example, forms, tables or direct debit mandates.</p>

```

I’m suggesting flipping the 2 lines, and slightly tweaking the examples of how this feature can be used.